app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -3,65 +3,74 @@
      data-caption-stt-enabled="<%=Settings.speech_to_text.enabled%>"
      data-caption-ocr-enabled="<%=Settings.ocr.enabled%>">
 
-<div class="row">
-    <div class="col">
-        <h1 class="h3">Start new job</h1>
-    </div>
-    <div class="col" style="text-align: right;">
-      <% if Settings.globus.enabled %>
-        <button class="btn btn-sul-dlss" data-globus-target="requestGlobusLink" data-action="click->globus#createDestination">Request Globus Link</button>
-      <% end %>
+    <div class="row">
```

Comment:
don't be fooled by this massive diff, it is literally me fixing an indentation problem, by indenting everything after the opening DIV .... there is one small change in this view, which is a adding target for the control we want to hide/show -- that's it